docs/pages/get-started/faq.md

```diff
@@ -410,83 +423,141 @@ Currently, messaging incurs no fee. As XMTP decentralizes, messaging between par
-
-There are no message storage and retrieval-related fees incurred by developers for building with the XMTP SDK.
-
 ## Security
 
 ### Has XMTP undergone a security audit?
 
+A security audit of [LibXMTP](https://github.com/xmtp/libxmtp) and its use of Messaging Layer Security (MLS) will begin in Q4 2024. The audit will be performed by [NCC Group](https://www.nccgroup.com/).
+
 The [XMTP JavaScript SDK](https://github.com/xmtp/xmtp-js) (`xmtp-js`) was verified by [CertiK](https://www.certik.com/company/about) on Feb 6, 2023. Read their [XMTP security assessment](https://raw.githubusercontent.com/xmtp/docs-xmtp-org/main/docs/pages/img/REP-final-20230207T000355Z.pdf).
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys to start using XMTP. XMTP uses these keys to establish a shared secret between the blockchain accounts. It then uses the shared secret to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
+XMTP uses a combination of Ethereum wallet signatures and MLS (Messaging Layer Security):
 
-To learn more about these keys, see [Key generation and usage in XMTP](/protocol/v2/key-generation-and-usage).
+1. A user starts with an Ethereum wallet and installs an XMTP-enabled app.
+
+2. The user creates an XMTP identity, which involves generating an identity key.
+
+3. For each app or device, an installation key is created and signed by the identity key.
+
+4. The installation key is used to create an MLS credential, which is signed by the Ethereum wallet.
 
-To learn more about invitation and message encryption, see [Invitation and message encryption with XMTP](/protocol/v2/invitation-and-message-encryption).
+5. The MLS credential contains the wallet address and a hash of the installation's signature verification key.
+
+6. For group chats, XMTP uses the MLS protocol to manage secure communication between multiple parties.
+
+For more details, see [Group chat concepts and protocols](/protocol/v3/group-chat#security-and-encryption) and [Identity in XMTP v3](/protocol/v3/identity)
+
+#### XMTP V2
+
+In V2, blockchain accounts sign and advertise a set of keys to start using XMTP. These keys are used to establish a shared secret between the blockchain accounts. The shared secret is then used to generate a key for encrypting an invitation, allowing the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
+
+For more details, see [Key generation and usage in XMTP v2](/protocol/v2/key-generation-and-usage) and [Invitation and message encryption with XMTP v2](/protocol/v2/invitation-and-message-encryption).
 
 ### Does each blockchain account address have a corresponding XMTP identity?
 
+Yes, each blockchain account address is associated with an XMTP identity.
```

Comment:
is this an accurate summary of how v3 works?

Reply:
Based on @richardhuaaa's review, I updated this section as follows: https://github.com/xmtp/docs-xmtp-org/pull/26/files#diff-0f5558e2ccc886a61098eac3e4020166d6c4dee87249fc7dcda8eba0f5bd1196R473